contracts/Registry.vy

```diff
@@ -1000,3 +1204,37 @@ def set_liquidity_gauges(_pool: address, _liquidity_gauges: address[10]):
             self.liquidity_gauges[_pool][i] = ZERO_ADDRESS
         else:
             break
+    self.last_updated = block.timestamp
+
+
+@external
+def set_pool_asset_type(_pool: address, _asset_type: uint256):
```

Comment:
Is this bloat since we have the `batch_set_pool_asset_type` fn?

Reply:
If the bytecode allows it, why not include? :shrug: